src/components/molecules/navbar/Navbar.tsx

```diff
@@ -11,6 +11,7 @@ const DefaultNavbar = () => {
     <Menu>
       <MenuItem label={'Login'} path={'/login'} />
       <MenuItem label={'Bli medlem'} path={'/registrer'} />
+      <MenuItem label={'Stillingsutlysninger'} path={'/jobs'} />
```

Comment:
I think "stillingsutlysninger" should only be under AuthNavbar as this only handles login and register